### DISCUSSION TITLE
Hide the Action bar

### DISCUSSION THREAD

**Discussion by ahaahr** [afreakyelf/Pdf-Viewer#40] [General]
Is there a way to hide the action bar in the top?

The docs say "Parent theme can be either Theme.PdfView.Light or Theme.PdfView.Dark or the one with no actionbar from the application". I am not sure what is meant by "from the application", but I tried the following:

`<style name="Theme.PdfView.SelectedTheme" parent="Theme.MaterialComponents.NoActionBar" />`

which causes this crash:

`java.lang.RuntimeException: Unable to start activity ComponentInfo{com.jabra.blueparrott/com.rajat.pdfviewer.PdfViewerActivity}: android.view.InflateException: Binary XML file line #10 in com.jabra.blueparrott:layout/activity_pdf_viewer: Binary XML file line #7 in com.jabra.blueparrott:layout/pdf_view_tool_bar: Error inflating class androidx.appcompat.widget.Toolbar`

I also tried this
`<style name="Theme.PdfView.SelectedTheme" parent="@style/Theme.PdfView.Light">
            <item name="windowActionBar">false</item>
            <item name="windowNoTitle">true</item>
        </style>`

which doesn't seem to have any effect.

I can change the back icon with the property `pdfView_backIcon`, so the styles I am using does have effect.

Thanks for your time :)

**Comment by afreakyelf** [comment]
Hi Thank you for pointing out. I have added a new attribute `showToolBar` in styles which is true by default. You can disable it by setting `false`. Changes will soon be out in a new release.
